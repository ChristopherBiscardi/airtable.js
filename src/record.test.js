import cases from "jest-in-case";
import Record from "./record";

const data = [{ table: {}, recordId: 5, recordJson: {} }];

describe("record", () => {
  cases(
    "new Record(table, recordId, recordJson)",
    ({ table, recordId, recordJson }) => {
      expect(new Record(table, recordId, recordJson)).toMatchSnapshot();
    },
    data
  );
});
