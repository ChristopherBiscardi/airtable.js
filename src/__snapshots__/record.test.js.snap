// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`record new Record(table, recordId, recordJson) case: 1 1`] = `
Record {
  "_rawJson": Object {},
  "_table": Object {},
  "destroy": [Function],
  "fetch": [Function],
  "fields": Object {},
  "id": 5,
  "patchUpdate": [Function],
  "putUpdate": [Function],
  "replaceFields": [Function],
  "save": [Function],
  "updateFields": [Function],
}
`;
